File: src/main/resources/static/js/service/planParamService.js
app.service('planParamService',function ($http) {
    this.findByDeviceIdAndPlanModeNo = function (deviceId,planModeNo,serverId) {
        return $http.get('../planParam/findByDeviceIdAndPlanModeNo?deviceId='+deviceId+"&planModeNo="+planModeNo+"&serverId="+serverId);
    };
});

File: src/main/java/com/shengxuan/speed/service/PlanParamService.java
package com.shengxuan.speed.service;

import com.shengxuan.speed.entity.pojo.PlanParam1;

import java.util.List;

public interface PlanParamService {

    /**
     * 根据设备id查询
     * @param deviceId
     * @return
     */
    List<PlanParam1> findByDeviceIdAndPlanModeNo(String deviceId,String planModeNo,int serverId);
}
